contributions.md

```diff
@@ -6,17 +6,17 @@ permalink: /contributions/
 
 # Contribute
 
-**Opening new issue?** Please read [issues guidelines]({{ site.baseurl }}{% link pages/issues.md %}) first.
+**Opening a new issue?** Please read our [issue guidelines]({{ site.baseurl }}{% link pages/issues.md %}) first.
 
-**Contributing code?** Check [code contribution guidelines]({{ site.baseurl }}{% link pages/code_contributions.md %}).
+**Contributing code?** Check the [code contribution guidelines]({{ site.baseurl }}{% link pages/code_contributions.md %}).
 
-**Investigating bugs?** [debugging commands]({{ site.baseurl }}{% link pages/debugging.md %}) can help.
+**Investigating bugs?** These [debugging commands]({{ site.baseurl }}{% link pages/debugging.md %}) can help.
 
 **Need help with git?** [git howto]({{ site.baseurl }}{% link pages/branch_management.md %}) might be what you're looking for.
 
 Contributing code or data to Cataclysm: Dark Days Ahead is easy - simply fork the repository on our [GitHub](https://github.com/CleverRaven/Cataclysm-DDA/), make your changes, and then send us a pull request.
 
-Cataclysm:Dark Days Ahead is released under the Creative Commons Attribution ShareAlike 3.0 license.  The code and content of the game is free to use, modify, and redistribute for any purpose whatsoever.  See https://creativecommons.org/licenses/by-sa/3.0/ for details.
+Cataclysm: Dark Days Ahead is released under the Creative Commons Attribution ShareAlike 3.0 license.  The code and content of the game is free to use, modify, and redistribute for any purpose whatsoever.  See [https://creativecommons.org/licenses/by-sa/3.0/](https://creativecommons.org/licenses/by-sa/3.0/) for details.
```

Comment:
For Markdown on github.com (like README.md), Github changes URLs into clickable links automatically, but apparently it doesn't do that for Github Pages websites. I think this will fix the issue, but I can't really test it myself.